helm/chaos-mesh/templates/webhookconfiguration.yaml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
rename this file to `webhook-configuration.yaml`?

Reply:
All done. Please take a look again.Thanks.